admin/app/helpers/workarea/admin/content_helper.rb

```diff
@@ -34,9 +34,10 @@ def storefront_content_preview_path(content)
       end
 
       def options_for_category(category_id)
-        return nil unless category_id.present?
+        model = Catalog::Category.where(id: category_id).first
+
+        return '' unless model.present?
```

Comment:
We can still return a valid HTML list of options here, if you do something like `return options_for_select({}) unless model.present?`

Reply:
that actually returns the same thing :)

```
[2] pry(main)> include ActionView::Helpers
=> Object
[3] pry(main)> options_for_select({})
=> ""